### PR TITLE
Chromium Chronicle 19

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Chromium Chronicle 19

cc: @mkamesam3

### CODE REVIEW

File: site/en/blog/chromium-chronicle-19/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+{% endColumn %}
+
+{% endColumns %}
+
+<!-- Row 2 -->
+{% Columns %}
+
+{% Column %}
+Use <a href="https://www.jetbrains.com/help/clion/searching-everywhere.html">Global Search</a>
+to search across all files and symbols in Chrome OS by tapping the *Shift* key
```
Comment (mihajlija) [inline site/en/blog/chromium-chronicle-19/index.md:39]:
```suggestion
to search across all files and symbols in Chrome OS by tapping the **Shift** key
```

File: site/en/blog/chromium-chronicle-19/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+alt: >
+  Chromium Chronicle image
+tags:
+  - chromium-chronicle
+---
+
+**Episode 19:** by Tom Hughes in Mountain View, CA (March, 2021)<br>
+[Previous episodes](/tags/chromium-chronicle/)
+
+When dealing with large codebases, it's useful to have the code indexed, so you can navigate across symbols. For Googlers, CodeSearch works for Google3 and Chromium, but doesn't yet support Chrome OS, and isn't available to non-Googlers. Don't despair, you can use the CLion IDE (**TODO** go/clion-for-chromeos **TODO**) to get this functionality, in addition to many more useful features.
```
Comment (mihajlija) [inline site/en/blog/chromium-chronicle-19/index.md:18]: I just checked go/clion-for-chromeos  and it's live so may want to remove the TODO

**Review by mihajlija — APPROVED** [review]

File: site/en/blog/chromium-chronicle-19/index.md (hunk 2 above)
Comment (petele) [reply site/en/blog/chromium-chronicle-19/index.md]: It is live, but it's internal only, and this is targeted at non-Google Chrome engineers.

**Comment by petele** [comment]
Thanks @mihajlija!

**Pull request merged**